### PR TITLE
chore: add wrapper for keycloak

### PR DESCRIPTION
**Pull request by 04kash**
## Description

Please explain the changes you made here.

## Which issue(s) does this PR fix

- Fixes #?

## PR acceptance criteria

Please make sure that the following steps are complete:

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary (requirement for new features)
- [ ] Add a screenshot if the change is UX/UI related

### CODE REVIEW

**Comment by 04kash** [comment]
This needs to be merged first: https://github.com/backstage/community-plugins/pull/1816

File: dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "janus-idp-backstage-plugin-keycloak-backend-dynamic",
-  "version": "3.1.0",
+  "name": "backstage-community-plugin-catalog-backend-module-keycloak-dynamic",
+  "version": "3.2.0",
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json:3]: why moving up to 3.2 and not 3.1.2?

File: dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json [hunk 2]
```diff
@@ -11,8 +11,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.32.4",
-    "pluginId": "janus-idp-keycloak-backend",
-    "pluginPackage": "janus-idp-keycloak-backend"
+    "pluginId": "backstage-community-plugin-catalog-backend-module-keycloak",
+    "pluginPackage": "backstage-community-plugin-catalog-backend-module-keycloak"
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json:15]: are you sure we want `backstage-community-` and not `@backstage-community/` or someting else like here?

https://github.com/janus-idp/backstage-showcase/blob/main/dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-pingidentity-dynamic/dist-dynamic/package.json#L14-L15

File: dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json [hunk 3]
```diff
@@ -11,8 +11,8 @@
   "backstage": {
     "role": "backend-plugin-module",
     "supported-versions": "1.32.4",
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json:13]:
```suggestion
    "supported-versions": "1.32.5",
```

File: dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json (hunk 1 above)
Comment (04kash) [reply dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json:3]: Had to add some changes from janus over to BCP, after that the latest version is 3.2.0

Comment (nickboldt) [reply dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json:3]: OK!

File: dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json (hunk 2 above)
Comment (04kash) [reply dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json:15]: Yeah, need to make changes to the `package.json` in community-plugins as well

Comment (nickboldt) [reply dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/dist-dynamic/package.json:15]: https://github.com/backstage/community-plugins/pull/1855 merged. Does that mean we need to bump this to 1.32.6 once it's re-released?

File: dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "janus-idp-backstage-plugin-keycloak-backend",
-  "version": "3.1.0",
+  "name": "backstage-community-plugin-catalog-backend-module-keycloak",
+  "version": "3.2.0",
```
Comment (nickboldt) [inline dynamic-plugins/wrappers/backstage-community-plugin-catalog-backend-module-keycloak-dynamic/package.json:3]:
```suggestion
  "version": "3.2.1",
```

File: yarn.lock [hunk 5]
```diff
@@ -2912,6 +2912,23 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@backstage-community/plugin-catalog-backend-module-keycloak@npm:3.2.0":
```
Comment (nickboldt) [inline yarn.lock:2915]:
```suggestion
"@backstage-community/plugin-catalog-backend-module-keycloak@npm:3.2.1":
```

File: yarn.lock [hunk 6]
```diff
@@ -2912,6 +2912,23 @@ __metadata:
...
+  version: 3.2.0
```
Comment (nickboldt) [inline yarn.lock:2916]:
```suggestion
  version: 3.2.1
```

**Review by nickboldt — APPROVED** [review]

**Pull request merged**